### ISSUE TITLE
CHROME: menuitem activedescendant not announced unless it is a *direct* descendant

### ISSUE THREAD

**Issue by brennanyoung** [nvaccess/nvda#9083]
### Steps to reproduce:
Using a menu pattern inspired by the W3C patterns, with menuitems focused by aria-activedescendant. Compare how NVDA announces the activedescendant when it is a direct descendant of the element where aria-activedescendant is set, and when it is 'deeper' in the tree.
Some example code [here](https://codepen.io/anon/pen/NEZRKR) - compare menu 1 and menu 2.
Tested in Chrome, IE11, FF

### Actual behavior:
With Chrome NVDA fails to announce the menu items where the menu items are more than one level deeper than the element where aria-activedescendant is set. (They must be a **direct** descendant of that element). Didn't test with Safari, but IE11 and FF do the right thing with both approaches. This may be a Chrome issue, rather than NVDA, but JAWS+Chrome handles this just fine.

### Expected behavior:
Given that aria-activedescendant refers to an id, which must be unique to the document, I would expect NVDA to announce the activedescendant even if it is many levels deeper than the place where aria-activedescendant is set. I would also expect it to announce even if the activedescendant is somewhere else in the tree, if aria-owns is also used. (Setting that attribute doesn't work either).

#### NVDA installed/portable/running from source:
installed
#### NVDA version:
2018.4
#### Windows version:
8.1

#### Name and version of other software in use when reproducing the issue:
Chrome 70.0.3538.110 (Official Build) (64-bit) (cohort: Stable)

#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA?
Yes, various 2018.3.x

**Comment by Adriani90** [comment]
the behavior is the same for me in Chrome 81 and Firefox 76.
Pressing enter on the Menu ABC, NVDA reports the first item focuse which is "a".
Pressing enter on the Menu PQR NVDA reports only "menu". Pressing down arrow in focus mode reveals that the Focus was on P but this was not reported.
I don't know if I am missunderstanding here something, but I think the browsers are exposing everything correctly here. If so, then this might be a bug in NVDA indeed.
cc: @jcsteh at least for Firefox and @aleventhal for Chrome.

**Comment by jcsteh** [comment]
This test case is wrong. aria-activedescendant gets set on menu2, but DOM focus remains on menuButton2. [As per the spec](https://w3c.github.io/aria/#aria-activedescendant), aria-activedescendant only has effect on the focused DOM element.
